src/index.html

```diff
@@ -9,6 +9,63 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <label class="image">
+        <a class="logo" href="localhost:8080">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </label>
+      <nav class="nav">
+        <ul class="list list__apple">
+          <a class="text text-1 is-active"
+            href="https://www.apple.com/"
+            >Apple
+          </a>
+        </ul>
+        <ul class="list">
+          <a class="text text-2"
+            href="https://www.samsung.com/"
+            >Samsung
+          </a>
```

Comment:
misuse of tags
![Screenshot 2022-10-28 at 15 09 38](https://user-images.githubusercontent.com/75496546/198583736-75f25358-04aa-4bc6-b101-f02ec189515b.png)
should be like this (`ul` – unordered list, `li` – list item)

Reply:
done